src/sentry/api/endpoints/organization_member/index.py

```diff
@@ -314,13 +316,16 @@ def post(self, request: Request, organization) -> Response:
         """
         Add or invite a member to an organization.
         """
-        if not features.has("organizations:invite-members", organization, actor=request.user):
+        assigned_org_role = request.data.get("orgRole") or request.data.get("role")
+
+        if assigned_org_role != "billing" and not features.has(
+            "organizations:invite-members", organization, actor=request.user
+        ):
```

Comment:
Inviting billing members would be equivalent to inviting owners on the developer plan.

I wasn't able to invite another owner on the developer plan, so that would also need to change to match the behaviour:

<img width="926" alt="Screenshot 2024-11-12 at 2 34 07 PM" src="https://github.com/user-attachments/assets/834dad20-9deb-4de3-8308-27ed5470d52f">